README.md

```diff
@@ -18,7 +18,7 @@ Kubernetes is:
 * **extensible**: modular, pluggable, hookable, composable
 * **self-healing**: auto-placement, auto-restart, auto-replication
 
-Kubernetes builds upon a decade and a half of experience at Google running production workloads at scale, combined with best-of-breed ideas and practices from the community.
+Kubernetes builds upon a [decade and a half of experience at Google running production workloads at scale](https://research.google.com/pubs/pub43438.html), combined with best-of-breed ideas and practices from the community.
```

Comment:
Hm, maybe "decade and a half of experience running production workloads at Google scale" flows a tiny bit better?